src/parquet/column/properties.h

```diff
@@ -77,9 +77,8 @@ class PARQUET_EXPORT ReaderProperties {
 
 ReaderProperties PARQUET_EXPORT default_reader_properties();
 
-static int64_t DEFAULT_PAGE_SIZE = 1024 * 1024;
+static int64_t DEFAULT_PAGE_SIZE = 64 * 1024 * 1024;
```

Comment:
Misread the double 1024 here. Thought it was only 1k, so I upped to mentally 64k. We should stay with 1M probably. Will revert this.